java-compiler-testing/src/test/java/io/github/ascopes/jct/tests/unit/filemanagers/JctFileManagerImplTest.java

```diff
@@ -0,0 +1,29 @@
+package io.github.ascopes.jct.tests.unit.filemanagers;
+
+import io.github.ascopes.jct.filemanagers.JctFileManager;
+import io.github.ascopes.jct.filemanagers.ModuleLocation;
+import io.github.ascopes.jct.filemanagers.impl.JctFileManagerImpl;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DisplayName("JctFileManager Tests")
+public class JctFileManagerImplTest {
+
+  @Test
+  @DisplayName("Tests whether we get instance of JctFileManager")
+  void testGettingJctFileManagerImplInstance() {
+    assertTrue(JctFileManagerImpl.forRelease("test") instanceof JctFileManagerImpl);
```

Comment:
cool, so as discussed on the issue, the AssertJ version of this would be something like

```java
// Given
var result = JctFileManagerImpl.forRelease("test"));

assertThat(result)
    .isInstanceOf(JctFileManagerImpl)
    .extracting(JctFileManagerImpl::getRelease)
    .isEqualTo("test");
```

Likewise for the throws test:

```java
assertThatThrownBy(() -> JctFileManagerImpl.forRelease(null))
    .isInstanceOf(NullPointerException.class)
    .hasMessage("...");
```

If you check out some of the existing test classes, you can see the style in which the tests are named and how stuff like `@DisplayName` is worded to keep things totally consistent! :-)